capture-sdk/sdk/src/main/res/layout/gc_fragment_analysis.xml

```diff
@@ -131,4 +132,13 @@
         </LinearLayout>
 
     </LinearLayout>
+
```

Comment:
[Request] Please add gc at the beginning of ids in capture-sdk

Reply:
This component was deleted